scalalib/src/Versions.scala

```diff
@@ -2,7 +2,7 @@ package mill.scalalib
 
 object Versions {
```

Comment:
And when we start to use `BuildInfo` module in our own build (see my comment to `MillModule`), we also might want to generate this file. _But this is only note, not a request, as that change is unrelated to the success of this PR._

Reply:
Ignore this comment as well, it's unrelated to this PR.